pkgs/tools/misc/btrfs-assistant/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, btrfs-progs
+, btrfs-snap
+, cmake
+, git
+, pkg-config
+, qtbase
+, qtsvg
+, qttools
+, util-linux
+, wrapQtAppsHook
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "btrfs-assistant";
+  version = "1.8";
+
+  src = fetchFromGitLab {
+    owner = "btrfs-assistant";
+    repo = "btrfs-assistant";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-Ay2wxDVue+tG09RgAo4Zg2ktlq6dk7GdIwAlbuVULB4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+    pkg-config
+  ];
+
+  buildInputs = [
+    btrfs-progs
+    btrfs-snap
+    qtbase
+    qtsvg
+    qttools
+  ];
+
+  propagatedBuildInputs = [ wrapQtAppsHook ];
+
+  postPatch = ''
+    substituteInPlace src/org.btrfs-assistant.pkexec.policy \
+      --replace '/usr/bin' "$out/bin"
+
+    substituteInPlace src/btrfs-assistant \
+      --replace 'btrfs-assistant-bin' "$out/bin/btrfs-assistant-bin"
+
+    substituteInPlace src/btrfs-assistant-launcher \
+      --replace 'btrfs-assistant' "$out/bin/btrfs-assistant"
+  '';
```

Comment:
@AndersonTorres  I'm not done with this PR, yet. But, in relation to what we were talking about in https://github.com/NixOS/nixpkgs/pull/247021 do you think we would rather explicitly change each of these files with a patch file or is this a use case that the substituteInPlace is acceptable?

Reply:
It looks acceptable. `substitute*` functions warn when the pattern does not find a match, quite the opposite of `sed`.